rcl_yaml_param_parser/src/parser.c

```diff
@@ -265,6 +265,21 @@ bool rcl_parse_yaml_file(
     return false;
   }
 
+
+  // Emitter for nested parameters
+  yaml_emitter_t  emitter;
+  success = yaml_emitter_initialize(&emitter);
+  if (0 == success) {
+    RCUTILS_SET_ERROR_MSG("Could not initialize the emitter");
+  }
+
+  size_t max_string_length = 100000;
+  size_t written_size = 0;
+  unsigned char nested_param_string_allocator[100000];
```

Comment:
```suggestion
  unsigned char nested_param_string_allocator[max_string_length];
```
Or is this a cpp feature and does not work for C code ?